cmd/bbolt/command_dump.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"go.etcd.io/bbolt/internal/guts_cli"
+)
+
+func newDumpCommand() *cobra.Command {
+	dumpCmd := &cobra.Command{
+		Use:   "dump <bbolt-file> pageid [pageid...]",
+		Short: "prints a hexadecimal dump of one or more pages of bbolt database.",
+		Args:  cobra.MinimumNArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Could we extract the args within `RunE()` and pass the args by name to `dumpFunc()` ?